.github/workflows/test_bioscrape.yml

```diff
@@ -0,0 +1,29 @@
+name: test_bioscrape
+on:
+  push:
+    branches: ["*","!master"]
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 4
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.10"]
```

Comment:
Can we do 3.10 and 3.7 here? Sandwich the versions?

Reply:
Yes. Done.